nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -87,9 +87,11 @@ FOR /F "usebackq eol=# tokens=1,* delims==" %%A in ("%NUXEO_CONF%") do (
   if "%%A" == "nuxeo.log.dir" set LOCAL_NUXEO_LOG_DIR=%%B
   if "%%A" == "nuxeo.tmp.dir" set LOCAL_NUXEO_TMP_DIR=%%B
   if "%%A" == "JAVA_OPTS" (
-    set __JAVA_OPTS=%%B
-    REM ***** Expand local string and replace $JAVA_OPTS with %LOCAL_JAVA_OPTS% *****
-    set LOCAL_JAVA_OPTS=!LOCAL_JAVA_OPTS! !__JAVA_OPTS:$JAVA_OPTS=%LOCAL_JAVA_OPTS%!
+    REM ***** Remove word "$JAVA_OPTS" from the line
+    REM ***** And append the remaining values to the existing options
+    set NEW_OPTS=%%B
+    set NEW_OPTS=!NEW_OPTS:$JAVA_OPTS=!
+    set LOCAL_JAVA_OPTS=!LOCAL_JAVA_OPTS!!NEW_OPTS!
```

Comment:
Compared to the original, you removed a space. Is it really inconsequential? Is it needed?